Fix adding classname when using SyntaxHighlighter block

renatho opened the pull request:
Fixes #123 and fixes #130

#### Changes proposed in this Pull Request:

* Fix adding classname when using Syntax Highlighter block

#### Testing instructions:

* Create a new Syntax Highlighter block in a post or page.
* In the sidebar, `Advanced` → `Additional CSS class(es)` add a class.
* Open the page on the frontend and check if the class was added.

Commit:
Add parse classname attribute to syntaxhighlighter block

gikaragia reviewed (approved):
So after testing this a bit I have found out that only the first class is included in the frontend.

The HTML that the plugin generates it is correct (it includes all classes) however the SyntaxHighlighter JS library modifies the HTML and it only adds the first class. It's supposed to [support multiple classes](http://alexgorbatchev.com/SyntaxHighlighter/manual/configuration/) however I wasn't able to make it work and reading its source code didn't help.

I think that we should merge this fix but leave issue #130 open, lower its priority and repurpose it to mention that multiple classes are not supported. Not sure if a fix is possible without modifying the dependency library though.

Commit:
Fix when block or shortcode uses more than one css class

renatho commented:
> So after testing this a bit I have found out that only the first class is included in the frontend.

Good catch!

I realized [here](http://alexgorbatchev.com/SyntaxHighlighter/manual/demo/class-name.html) that he uses single quotes in the `class-name` attribute. So I tested and it worked with multiple classes \o/ 

Fixed here: https://github.com/Automattic/syntaxhighlighter/pull/138/commits/53786a3e2ce5b6b77c5259d39b3dbd49f9f57e1a

The name of the method where I fixed it is `shortcode_callback`, but the `render_block` also calls this method. Alex leave this comment in the `render_block` method:

`Renders the content of the Gutenberg block on the front end using the shortcode callback. This ensures one source of truth and allows for forward compatibility.`

gikaragia commented:
Nice! Didn't thought of using quotes :)
